gordon_janitor_gcp/cloud_dns.py

```diff
@@ -24,9 +24,12 @@
 
 .. code-block:: pycon
 
+    >>> from gordon_janitor_gcp import auth
     >>> keyfile = '/path/to/service_account_keyfile.json'
+    >>> auth_client = auth.GoogleAuthClient(keyfile=keyfile)
+    >>> creds = auth_client.creds
```

Comment:
(I didn't notice it last time) I'm looking at this example, the docstring for `__init__`, and the [google credential docs](http://google-auth.readthedocs.io/en/latest/reference/google.auth.credentials.html#google.auth.credentials.Credentials) and it seems to me that AIOGoogleDNSClient should be initialized with `auth_client` instead of `auth_client.creds`. The AIOGoogleHTTPClient expects to get a session object from the `auth_client.creds` object, but I don't think that creds, an instance of `google.auth.credentials.Credentials`,  have a `_session` attribute, but `auth_client` has both `creds` and `_session` attributes.

Reply:
good catch - thank you!